.github/workflows/test.yml

```diff
@@ -39,12 +41,9 @@ jobs:
   integration:
     name: Integration
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        postgresql-version: [11, 12, 13]
     services:
       postgres:
-        image: postgres:${{ matrix.postgresql-version }}
+        image: postgres:latest
```

Comment:
Will using `:latest` for testing like this lead to us testing an environment that doesn't match what is being run? Wouldn't testing a matrix of versions be safer?

Reply:
Well. mreg-cli doesn't interface directly with Postgres, mreg does that. So the question becomes, what version of Postgres is mreg using. Both the [test workflow](https://github.com/unioslo/mreg/blob/master/.github/workflows/test.yml#L17) and the [container image workflow](https://github.com/unioslo/mreg/blob/master/.github/workflows/container-image.yml#L36) use `postgres:latest` for testing. What people use in production environments may vary.

The purpose of mreg-cli is to interface with mreg. The way I see it, if mreg doesn't work because of Postgres incompatibilities, thats a problem with mreg not mreg-cli.

Reply:
Fair enough, and besides, if `:latest` actually breaks something in testing, it's the best place to find it early.